tests/arm64/Tests.lst

```diff
@@ -1840,20 +1840,6 @@ Expected=100
 MaxAllowedDurationSeconds=600
 Categories=Pri0;EXPECTED_PASS
 HostStyle=Any
-[badldsfld_il_d.exe_263]
-RelativePath=JIT\Directed\coverage\importer\Desktop\badldsfld_il_d\badldsfld_il_d.exe
-WorkingDir=JIT\Directed\coverage\importer\Desktop\badldsfld_il_d
-Expected=100
-MaxAllowedDurationSeconds=600
-Categories=Pri0;EXPECTED_FAIL
```

Comment:
I don't think we necessarily adds more tags to tests that don't even run in the baseline given the fact we use the fixed set of tests. And also we don't want to update two places -- issues.targets and here when someone wants to update it. Ideally/eventually we will migrate syncing test population when PR is sent . At minimum we should sync the test population in a regular fashion like a week or so from automation so that we can get up-to-date new/exclusion tests.